conftest.py

```diff
@@ -12,40 +12,18 @@
 """
 
 import os
-import os.path as osp
-import sys
-
-import pylsp_utils
 
 # ---- To activate/deactivate certain things for pytest's only
 # NOTE: Please leave this before any other import here!!
 os.environ['SPYDER_PYTEST'] = 'True'
 
-
-# ---- Detect if we're running in CI
-# Note: Don't import from spyder to keep this file free from local
-# imports.
-running_in_ci = bool(os.environ.get('CI'))
-
-
-# ---- Handle subrepos
-# NOTE: Please don't move this from here!
+# ---- Update PyLSP
 # When running in CI, subrepos are installed in the env.
-if not running_in_ci:
-    HERE = osp.dirname(osp.abspath(__file__))
-    DEPS_PATH = osp.join(HERE, 'external-deps')
-    i = 0
-    for path in os.listdir(DEPS_PATH):
-        external_dep_path = osp.join(DEPS_PATH, path)
-        sys.path.insert(i, external_dep_path)
-        i += 1
-
-    # Remove previous local PyLSP installation.
-    pylsp_utils.remove_installation()
-
-    # Install PyLS locally.
-    pylsp_utils.install()
-
+if not os.environ.get('CI'):
+    from install_dev_repos import REPOS, install_repo
+    if REPOS['python-lsp-server']['editable']:
+        # Installed in editable mode -> reinstall
+        install_repo('python-lsp-server', editable=True)
```

Comment:
```suggestion
```

Likewise, this is a pretty big cost to pay every test run, when it only matters for very specific tests, when python-lsp-server is updated to a new breaking version, that changes its main/plugin entrypoints at a very high level (which should be quite uncommon), and can be solved by just re-running `install_dev_repos.py` if its needed.